compose.yaml

```diff
@@ -28,12 +28,13 @@ networks:
 
 services:
   openc3-minio:
+    user: "${OPENC3_USER_ID:-1001}:${OPENC3_GROUP_ID:-1001}"
```

Comment:
It's going to work out to `0:-1001:0:-1001` normally or like `501:-1001:501:-1001` for rootless (whatever you user id is). Is this mapping one to the other or is `1001` a default if the variables aren't set.

Reply:
1001 is default if the variable is not set.    It will be 0 for podman, and 501 for mac, and 1000 for the first user on Ubuntu, etc.